hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -578,6 +587,46 @@ public InputStream decompress(InputStream is) {
          * @param blob the blob to publish
          */
         void publish(HollowProducer.Blob blob);
+
+        /**
+         * Publish the headerBlob specified to this publisher's header blobstore.
+         * <p>
+         * {@link BlobStager#openHeader(long)} on this publisher.
+         *
+         * @param headerBlob the header blob to publish
+         */
+        void publish(HollowProducer.HeaderBlob headerBlob);
```

Comment:
We should provide a default implementation for backwards compatibility

Reply:
ACK